src/avkit.cs

```diff
@@ -134,6 +139,16 @@ interface AVPlayerViewController {
 		[Export ("updatesNowPlayingInfoCenter")]
 		bool UpdatesNowPlayingInfoCenter { get; set; }
 
+		[iOS (11, 0)]
```

Comment:
Minor comment `[iOS (11,0)]` is prettier (note: you're sometimes not adding a space in this diff).